chart/openfaas-oidc-proxy/README.md

```diff
@@ -74,59 +73,19 @@ EOF
 kubectl apply -f issuer-prod.yaml
 ```
 
-Create an ingress record for the openfaas-oidc-proxy:
-
-```bash
```

Comment:
I missed that this was in the README file somehow.

I thought we'd skipped the instructions which is why I asked for you to add it to the chart. I think it's better in the chart anyway - so thank you for adding it.